Scan/utkscan/CMakeLists.txt

```diff
@@ -34,16 +30,65 @@ if(UTKSCAN_VERBOSE)
     add_definitions(-D VERBOSE)
 endif(UTKSCAN_VERBOSE)
 
-#------------------------------------------------------------------------------
-
 #Check if GSL is installed
 if(USE_GSL)
     find_package(GSL REQUIRED)
     add_definitions("-D usegsl")
 endif(USE_GSL)
 
+#------------------------------------------------------------------------------
+
+#Add the includes from ScanLib to our tree
+include_directories(include ../ScanLib/include)
+
+#Add the local include directories to the build tree
+include_directories(include analyzers/include)
+include_directories(include core/include)
+include_directories(include experiment/include)
+include_directories(include processors/include)
```

Comment:
Not all of it, but it's such a mixed bag that it's easier than cherry picking which ones need it and which do not. Remember that we have a mess of circular dependencies here.